client/wfprev-war/src/main/angular/src/app/components/edit-project/project-details/fiscal-year-projects/fiscal-year-projects.component.html

```diff
@@ -12,7 +12,7 @@
             <div *ngFor="let fiscal of projectFiscals" class="fiscal-item">
                 <div class="fiscal-title-row">
                     <span class="section-1">{{fiscal.projectFiscalName}}</span>
-                    <span class="section-2"> Fiscal Year: 
+                    <span class="section-2"> Fiscal Activity:
```

Comment:
same here, I think we should keep "Fiscal Year" for the year label

Reply:
Reverted to original